packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -313,15 +319,23 @@ const AccountTransferForm = observer(
                 return internal_remaining_transfers?.available;
             };
 
-            remaining_transfers = getRemainingTransfers();
-
-            const hint =
-                remaining_transfers && Number(remaining_transfers) === 1
-                    ? localize('You have {{number}} transfer remaining for today.', { number: remaining_transfers })
-                    : localize('You have {{number}} transfers remaining for today.', { number: remaining_transfers });
-            setTransferToHint(hint);
+            let hint_text;
+            // flag 'open_order_position_status' does not exist in mt5_login_list, @deriv/api-types yet
+            if (mt5_login_list.find(account => account?.login === selected_to.value)?.open_order_position_status) {
```

Comment:
the `mt5_login_list` array will have `open_order_position_status` set to true. since BE informed this flag will be optional, i think in the false case, the key won't exist in the response.
let me know if this didn't answer your question.